mitmproxy/eventsequence.py

```diff
@@ -70,4 +70,4 @@ def event_sequence(f):
             yield "tcp_error", f
         yield "tcp_end", f
     else:
-        raise NotImplementedError
+        raise ValueError
```

Comment:
Wouldn't TypeError be more appropriate?
(this should also raise an instance, so let's add parantheses)